event_location_manager/models/event_event.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+# @authors: Alexander Ezquevo <alexander@acysos.com>
+# Copyright (C) 2016  Acysos S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning
+from datetime import datetime, timedelta
+
+
+DFORMAT = "%Y-%m-%d %H:%M:%S"
```

Comment:
Take a look: https://github.com/OCA/OCB/blob/8.0/openerp/tools/misc.py#L831
You can import it using
```python
from openerp.tools import DEFAULT_SERVER_DATETIME_FORMAT as DFORMAT
```